app/src/main/java/app/wefridge/wefridge/exceptions/InternetUnavailableException.kt

```diff
@@ -0,0 +1,8 @@
+package app.wefridge.wefridge.exceptions
+
+class InternetUnavailableException: Exception {
+    constructor() : super("Internet unavailable")
+    constructor(message: String) : super(message)
+    constructor(message: String, cause: Throwable) : super(message, cause)
+    constructor(cause: Throwable) : super(cause)
```

Comment:
Are these lines necessary?

Reply:
Currently not, but it's convention to support all constructors (AFAIK).

Reply:
But they are the same as in the super class

Reply:
Oh, right. I will fix that

Reply:
Turns out, that they are not available anymore when I remove them, even though they are defined in the Exception class... I think it would be better to leave them there, unless anyone of us has a better idea / suggestions?

Reply:
Sure leave them there